src/TrackPanel.cpp

```diff
@@ -735,11 +742,62 @@ void TrackPanel::OnMouseEvent(wxMouseEvent & event)
          }
       } );
    }
+    
+    if(event.RightDown())
+    {
+        wxWindow *parent = wxWindow::FindFocus();
+        if(!parent)
+           parent = &GetProjectFrame( *GetProject() );
+        
+        if(parent)
+        {
+            wxMenu contextMenu;
+            contextMenu.Bind(wxEVT_MENU,
+               [this]( wxCommandEvent &event ){
+                  OnContextMenu( /*GetProject(),*/ event ); }
+            );
+            
+            contextMenu.Append(kContextMenuItemID_AddMonoTrack, XO("Add mono track").Translation());
+            contextMenu.Append(kContextMenuItemID_AddStereoTrack, XO("Add stereo track").Translation());
+            contextMenu.Append(kContextMenuItemID_AddLabelTrack, XO("Add label track").Translation());
+            contextMenu.AppendSeparator();
+            contextMenu.Append(kContextMenuItemID_ExportProject, XO("Export project\tCtrl+E").Translation());
```

Comment:
You have a hard-coded menu shortcut, which we now know is wrong.  You correct this in a later commit.  I don't want PRs past the draft stage that still keep a history of errors and corrections.